doc/SensorFusion/SRS/srs_eng.md

```diff
@@ -0,0 +1,385 @@
+
+  - The ZumoComSystem shall receive odometry from the Zumo32U4 via USB (using the RadonUlzer firmware).
+  - The ZumoComSystem may receive IMU data from the Zumo32U4 via USB.
+  - The ZumoComSystem shall receive absolute position, orientation, and velocity from the SpaceShipRadar via MQTT.
+
+- **Sensor Fusion**
+
+  - The ZumoComSystem shall fuse the received data using appropriate sensor fusion techniques. The data sources include:
+    - Odometry data (from Zumo32U4)
+    - (IMU data (from Zumo32U4))
+    - External position data (from SpaceShipRadar via MQTT)
+
+- **Data Publishing**
+
+  - The ZumoComSystem shall publish the calculated pose and velocity via MQTT for use by other systems.
+
+- **Simulation and Real-World Operatuion**
+
+  - The ZumoComSystem shall receive absolute position, orientation, and velocity from the SpaceShipRadar via MQTT.
+  - The software shall be capable of running in both the Webots simulation environment and on real hardware.
+
+- **Extensibility**
+  - The software architecture shall be modular to allow for future extensions, such as adding more sensor sources (e.g., magnetometer) or alternative sensor fusion methods.
+
+> System Overview Diagram (to be added later)
+
+## 2.3 Assumptions and Dependencies
+
+### 2.3.1 Assumptions
+
+| ID   | Type       | Description                                                 | Status                  |
+| ---- | ---------- | ----------------------------------------------------------- | ----------------------- |
+| A-01 | Assumption | The ZumoComSystem receives odometry minimally every 100ms.  | open                    |
+| A-02 | Assumption | The ZumoComSystem receives IMU data minimally every 100ms   | needs to be implemented |
+| A-03 | Assumption | A MQTT broker is available and reachable by all components. | open                    |
+
+### 2.3.2 Dependencies
+
+| ID   | Type       | Description                                                                       | Status |
+| ---- | ---------- | --------------------------------------------------------------------------------- | ------ |
+| D-01 | Dependency | Use of external software libraries: RadonUlzer, DroidControlShip, SpaceShipRadar. | active |
+| D-02 | Dependency | Simulation environment _Webots_ is provided and compatible with RadonUlzer/DCS.   | ok     |
+| D-03 | Dependency | Communication between Zumo32U4 and ZumoComSystem is reliably available.           | ok     |
+
+# 3. Requirements
+
+## 3.1 External Interface Requirements
+
+### 3.1.1 User Interfaces
+
+| ID     | Description                                                                                                   | Verified by | Implemented by |
+| ------ | ------------------------------------------------------------------------------------------------------------- | ----------- | -------------- |
+| UI-101 | The ZumoComSystem shall publish fused localization data (pose and velocity) via MQTT topic `zumo/<id>/fusion` |             |
```

Comment:
I think the definition of the topics at this level (requirements-level) is a bit too specific. But we can leave them here for the time being